### DISCUSSION TITLE
In product page, add a new field for Product price (tax incl.) in Specific Prices

### DISCUSSION THREAD

**Discussion by alfonsolr09** [PrestaShop/PrestaShop#40166] [Ideas]
#### Is your feature request related to a problem?

sometimes it is a bit heavy to calculate the price to make the price to your liking

#### Describe the solution you'd like

add box to apply price rule on a price with tax included

#### Additional context

![image](https://user-images.githubusercontent.com/62474824/106978924-8a02f080-675d-11eb-8c69-7d976a677455.png)

- the box on the left currently exists and is used to set a price excluding taxes.
- the box on the right could be implemented and would serve as a price with taxes included.

obviously only one of them could be used and the other would be calculated automatically as happens here
![image](https://user-images.githubusercontent.com/62474824/106979160-05fd3880-675e-11eb-85fb-c7321efdcf83.png)

**Comment by florine2623** [comment]
Hi @alfonsolr09 ,
Thanks for the suggestion.
Please stay tuned as I ping our product-team !

**Comment by florine2623** [comment]
Ping @PrestaShop/product-team ,

Do you think adding a new field for "**Product price (tax incl.)**", in product page > Price tab > Specific Prices > Impact on Price, would be possible ?

Thanks!
